docs/TPM_with_LPC_protocol.md

```diff
@@ -0,0 +1,202 @@
+## TPM module with "Low Pin Count Protocol" build on SoC
+
+### Introduction
+
+The main goal of this document is showing the current state of development of TPM
+module with "LPC Protocol" based on SoC (System on Chip), and also documenting
+work carried out so far.
+
+## Motivation for doing this project
+
+We stiil see the necessity of making TPM module with "Low Pin Count" protocol.
+There is still many computers on the market (especcialy servers) which have
+mother-boards supporting LPC protocol.
+We considered possible hardware platforms in the past in this document:
+
+["Target TPM hardware interfaces"](https://lpntpm.lpnplant.io/hardware_interfaces/#low-pin-count-interface)
+
+Outside of fast "SPI Bus" and even "I2C" protocol which are relatively easy to
```

Comment:
@mgabryelski1 We should use [backticks](https://en.wikipedia.org/wiki/Backtick) in markdown, not qutoation marks here